js/todos.js

```diff
@@ -0,0 +1,37 @@
+var App = new (Backbone.View.extend({
+    Models: {},
+    Views: {},
+    Collections: {},
+    template: _.template(
+        '<div class="main-container">' +
+        '<div class="todo-list-container"><h1>ToDo List</h1>' +
+        '<div id="app"></div>' +
+        '</div></div>' 
+    ),
+
+    render: function () {
+        this.$el.html(this.template());
+        return this;
+    },
+    start: function () {
+        this.$el.append(this.render().el);
+
+        // Initialize some data
+        App.Todos = new App.Collections.Todos();
+        App.Todos.add({ description: 'My First Todo' });
+        App.Todos.add({ description: 'The other Todo' });
```

Comment:
Yeah, these three lines specifically I would do outside the view and pass them in as the view's `collection` property. So:

```
        const todosCollection = new App.Collections.Todos([{
            { description: 'My First Todo' },
            { description: 'The other Todo' }
        }]);
       const appView = new AppView({ collection: todosCollection });
```

This makes it easier to test, because you can boot up an AppView with nothing but an empty collection (dependency injection).